project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject bugle-forms "0.1.0-SNAPSHOT"
+(defproject bugle-forms "latest"
```

Comment:
Let's version the app using practices outlined in https://semver.org/.

Reply:
Okay. How exactly would this work, btw? Our current CI/CD pipeline deploys every commit to `master` into prod, which makes every commit to it a new release.

So I suppose every merge to master should be accompanied by a new tag as well? It seems a bit weird to me somehow. Would this mean that every PR I make to `master` must edit this release number as well? That would seem quite hard to manage and keep a track of.

What do you think of changing the CI so that a production deploy only occurs on every tag push? That way the releases could be more controlled?

Reply:
> What do you think of changing the CI so that a production deploy only occurs on every tag push, instead of every commit?
This would be my preference as well.

Although we could simply deploy every commit to master into production, it'd be nicer to keep track of releases with versioning in order to make it easy to rollback or debug a particular release. Does that make sense?

Reply:
Okay. I'll switch to using semver, with deploys happening only when we push tags.